run_bin.go

```diff
@@ -210,3 +211,21 @@ func haveTestsToRun(output string) bool {
 	prefix := "testing: warning: no tests to run"
 	return !strings.HasPrefix(output, prefix)
 }
+
+func pipeInputToStdin(cmd *exec.Cmd, input string) {
```

Comment:
The most generic approach would be a callback function with the command. It's not clear if this is really necessary, or if you can be more specific like just passing a stdin string (or better, io.Reader object).

Something like

```
RunBinary(binPath, "TestBincoverRunMain", []string{}, tt.args, func(cmd *exec.Cmd) {
    pipeInputToStdin(cmd, stdinInput)
})
```

But I noticed that you sometimes need to do things to `cmd` before calling `exec` and sometimes after. So you may need multiple callbacks.

But you could expose them as part of the Builder object if you decide you need them.